ci/docker/Dockerfile.build.ubuntu

```diff
@@ -132,9 +132,17 @@ RUN cd /usr/local/src && \
     tar xf apache-rat-0.13-bin.tar.gz
 
 # Python & cmake
+# We need to remove NumPy from requirements for ubuntu18.04
+# because ubuntu18.04 has only python3.6, which is not supported 
+# for NumPy >= 1.20.0
```

Comment:
Ubuntu 18.04 is only used by the tensorrt image and only because in December 2020 NVidia hadn't made tensorrt available for 20.04 yet. (https://github.com/apache/incubator-mxnet/commit/afc76b0f82839dfe00f7cab04be6de3df94564fe)

Instead of going throgh the python3.6 trouble, I think you can just update https://github.com/apache/incubator-mxnet/blob/16fed6ed46c84137853ac47a019c370d2605e832/ci/docker/docker-compose.yml#L101-L110 and https://github.com/apache/incubator-mxnet/blob/16fed6ed46c84137853ac47a019c370d2605e832/ci/docker/Dockerfile.build.ubuntu#L164-L170 to ubuntu 20.04